### PR TITLE
update 1.16.5

### PR DESCRIPTION
**Pull request by avasomething**
Changelog:
- bump version number
- specify perl versions (and to ignore run exports from it)
- fix license version
- switch URLs to https
- replaced our shebang fix with conda-forge's to resolve failing aarch64 build

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
 
 test:
   commands:
     - automake --help
 
 about:
-  home: http://www.gnu.org/software/automake/
-  license: GPL 3
+  home: https://www.gnu.org/software/automake/
+  license: GPL-2.0-only
```
Comment (ryanskeith) [inline recipe/meta.yaml:35]: Note that the README file states: 
"This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2, or (at your option)
any later version."

So the source code is a bit ambiguous.  Not sure what the right answer here BUT the COPYING file is pretty clear.

**Review by ryanskeith — APPROVED** [review]
I think this looks good to me.  I still wonder about the license ambiguity I stated but I think what you have done is fair.

**Review by AndrewVallette — APPROVED** [review]
The `license` you selected looks right to me 👍

**Review by varlackc — COMMENTED** [review]
LGTM

**Review by varlackc — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**